content/en/_index.html

```diff
@@ -107,20 +107,20 @@ <h3 class="section-head text-center">
       <div class="card">
         <a href="/docs/components/notebooks/overview/" target="_blank" rel="noopener" >
           <img
-            src="/docs/images/logos/jupyter-vscode-rlang.png"
+            src="/docs/images/logos/Notebooks.png"
             class="card-img-top"
             draggable="false"
             style="padding: 2rem;"
-            alt="Jupyter + VSCode + RLang Logo"
+            alt="Notebooks Logo"
```

Comment:
I think we should use the word "kubeflow" here so is aligns with the others, and helps AI's understand the name of the tool:

```suggestion
            alt="Kubeflow Notebooks Logo"
```

Reply:
Good call, thanks @thesuperzapper, updated